requirements/edx/base.txt

```diff
@@ -25,7 +25,7 @@ git+https://github.com/mitodl/edx-sga.git@6b2f7aa2a18206023c8407e2c46f86d4b4c3ac
 git+https://github.com/edx/xblock-lti-consumer.git@v1.1.8#egg=lti_consumer-xblock==1.1.8
 git+https://github.com/edx/MongoDBProxy.git@25b99097615bda06bd7cdfe5669ed80dc2a7fed0#egg=MongoDBProxy==0.1.0
 -e .
-git+https://github.com/edx/edx-ora2.git@2.1.18#egg=ora2==2.1.18
+git+https://github.com/edx/edx-ora2.git@2.1.18#egg=ora2==2.1.19
```

Comment:
This will need to be updated to use the 2.1.19 tag once https://github.com/edx/edx-ora2/pull/1105 is merged and the new tag created.